cdc/processor/tablepb/table.pb.go

```diff
@@ -179,7 +179,7 @@ type Stats struct {
 	// Number of captured regions.
 	RegionCount uint64 `protobuf:"varint,1,opt,name=region_count,json=regionCount,proto3" json:"region_count,omitempty"`
 	// The current timestamp from the table's point of view.
-	CurrentTs Ts `protobuf:"varint,2,opt,name=current_ts,json=currentTs,proto3,casttype=Ts" json:"current_ts,omitempty"`
+	CurrentTs Ts `protobuf:"varint,2,opt,name=current_ts,json=currentTs,proto3,casttype=Ts" json:"current_ts,omitempty"` // Deprecated: Do not use.
```

Comment:
Why will this pr change cdc file?

Reply:
comes from https://github.com/pingcap/tiflow/commit/1b026f6b75b016208122cf56052e80bac8ba8f44#diff-c11c9ab601e606fc9e796253fc7ff93df630ae6eb29ec9b41762b923cdcb62d8 without regenerate protocol buffers